src/content/docs/en/reference/api-reference.mdx

```diff
@@ -399,6 +399,47 @@ Use [`Astro.url`](#astrourl) to construct your own canonical URL.
 
 The [canonical URL][canonical] of the current page.
 
+### `Astro.rewrite()`
+
+<p><Since v="4.13.0" /></p>
+
```

Comment:
As suggested by @sarah11918 in #8917, we should add the type of the function:

```suggestion

**Type:** `(rewritePayload: string | URL | Request) => Promise<Response>`

```

I replaced the [`RewritePayload` type](https://github.com/withastro/astro/blob/208f7f6470e873c715806e006747d0bd6351c3f9/packages/astro/src/%40types/astro.ts#L2818) (the parameter) by its value to make it more clear by reading the documentation (even if the possible values ​​are detailed below).

Reply:
Thank you! Will ask @ematipico to confirm this!